stacks-signer/src/client/stackerdb.rs

```diff
@@ -0,0 +1,197 @@
+use blockstack_lib::burnchains::Txid;
```

Comment:
Copyright comment block please

Reply:
Done. Added to ever file in stacks-signer now.